### ISSUE TITLE
Amazing project,  Quadriped support?

### ISSUE THREAD

**Issue by 99bits** [jasongzy/Make-It-Animatable#2]
Thank you for your incredible work and awesome project.
Would this work for Quadriped also?  If not, what would it take to extend the project to support Quadriped?

Cheers

**Comment by jasongzy** [comment]
Thanks for your interest!
Currently, quadruped animals are not supported, but that is basically a data issue.
The proposed framework should be able to work for quadruped animals if we can obtain a high-quality training dataset of quadruped meshes with a consistent skeleton topology.

**Comment by 99bits** [comment]
Amazing to know that the framework supports other types of models,.which makes it much more flexible.  
What would the training data need to look like, and how many models and poses are needed?

**Comment by jasongzy** [comment]
Our model is trained on **90** rigged character meshes and **2k** pose sequences from a dataset collected from [Mixamo](https://www.mixamo.com/), so you can find exemplar training data on Mixamo (all freely available).
Specifically, what we exactly need for training is: **posed meshes** (as inputs) and the correponding **blend weights** + **bone positions** + **pose-to-rest transformations** (as ground-truth labels).

**Comment by 99bits** [comment]
I presume that all of the training data has been from using mixamo ready made models and animations on their website. This is a great idea.  I need to understand in more detail what the training data format looks like.  As for the  blend weights, does it have to be measured with the bone at Tpose or in all poses.

Is it possible to get a small sample of what the training data set looks like?  This way I can try to duplicate it.

**Comment by 99bits** [comment]
Super helpful to get this kind of in depth instructions.  The more detailed the better so that I can follow your guidelines.  Looking forward to seeing the code for the project when it's posted.  

Thanks again
